content/en/hosting/4.x/production/docker/_index.md

```diff
@@ -1,17 +1,160 @@
+                  |-- compose/
+                  |-- certs/
+                  |-- couchdb/
+                  |-- upgrade-service/
+```
+
+By calling this `mkdir` commands:
+
+```shell
+mkdir -p /home/ubuntu/cht/{compose,certs,upgrade-service,couchdb}
+```
+
+1. `compose` - docker-compose files for cht-core and CouchDB
+2. `certs` -  TLS certificates directory
+3. `upgrade-service` - where docker-compose file for the upgrade-service
+4. `couchdb` - the path for the docker-compose file of the upgrade-service (not used in multi-node)
+
+## Download required docker-compose files
+
+The following 3 `curl` commands download CHT version `4.11.0` compose files, which you can change as needed. Otherwise, call:
+
+```shell
+cd /home/ubuntu/cht/
+curl -s -o ./compose/cht-core.yml https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:4.11.0/docker-compose/cht-core.yml
+curl -s -o ./compose/cht-couchdb.yml https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:4.11.0/docker-compose/cht-couchdb.yml
+curl -s -o ./upgrade-service/docker-compose.yml https://raw.githubusercontent.com/medic/cht-upgrade-service/main/docker-compose.yml
+```
+
+## Prepare Environment Variables file
+
+Prepare a `.env` file by running this code:
+
+```sh
+sudo apt install wamerican
+uuid=$(uuidgen)
+couchdb_secret=$(shuf -n7 /usr/share/dict/words --random-source=/dev/random | tr '\n' '-' | tr -d "'" | cut -d'-' -f1,2,3,4,5,6,7)
+couchdb_password=$(shuf -n7 /usr/share/dict/words --random-source=/dev/random | tr '\n' '-' | tr -d "'" | cut -d'-' -f1,2,3,4,5,6,7)
+cat > /home/ubuntu/cht/upgrade-service/.env << EOF
+CHT_COMPOSE_PROJECT_NAME=cht
+COUCHDB_SECRET=${couchdb_secret}
+DOCKER_CONFIG_PATH=/home/ubuntu/cht/upgrade-service
+COUCHDB_DATA=/home/ubuntu/cht/couchdb
+CHT_COMPOSE_PATH=/home/ubuntu/cht/compose
+COUCHDB_USER=medic
+COUCHDB_PASSWORD=${couchdb_password}
+COUCHDB_UUID=${uuid}
+# <ADD_SERVERNAME_TO_HTTP_AGENT>: (optional, default: false, configures: api) Adds 'servername' to HTTP agent for certificate issues in receiving traffic when proxying HTTPS->HTTP in SNI environments with external TLS termination. 
+# See 'tls.connect(options[, callback])' (https://nodejs.org/api/tls.html). May resolve 'ERR_TLS_CERT_ALTNAME_INVALID' error.
+# ADD_SERVERNAME_TO_HTTP_AGENT=true
+# <PROXY_CHANGE_ORIGIN>: (optional, default: false, configures: api) See http-proxy (https://www.npmjs.com/package/http-proxy#options). Sets 'changeOrigin' to 'true' for HTTP clients. For certificate issues in proxying HTTP->HTTPS
+# PROXY_CHANGE_ORIGIN=true
+EOF
+```
+
+Note that secure passwords and UUIDs were generated on the first four calls and saved in the resulting `.env` file. Uncomment optional items if required.
+
+## Launch containers
+
+{{% alert title="Note" %}}
+This section has the first use of `docker compose`.  This should work, but you may need to use the older style `docker-compose` if you get an error `docker: 'compose' is not a docker command.`.
+{{% /alert %}}
```

Comment:
Let's just make sure they're on the right version up front, either in the requirements doc, or by getting them to run `docker -v`

Reply:
ah - good call! fixed in b82f6647